src/conditions/BUILD

```diff
@@ -84,6 +84,15 @@ config_setting(
     visibility = ["//visibility:public"],
 )
 
+config_setting(
+    name = "linux_loongarch64",
+    constraint_values = [
+        "@platforms//os:linux",
+        "@platforms//cpu:loongarch64",
```

Comment:
This doesn't exist yet.   You'll want to start in github.com/bazelbuild/platforms and add a new architecture to the cpu list there.